src/extension/content-script/shouldInject.js

```diff
@@ -39,3 +46,17 @@ function documentElementCheck() {
   }
   return true;
 }
+
+function checkBlocklist() {
+  try {
+    const currentUrl = window.location.pathname;
+    console.log("shouldinject currentUrl, ", currentUrl);
+    const isBlocked = msg.request("getBlocklist", {
+      domain: currentUrl,
+      host: currentUrl,
+    });
+    console.log("2shouldinject isBlocked? ", isBlocked);
+  } catch (e) {
+    if (e instanceof Error) console.log(e.message);
```

Comment:
If this stays it could be switched to `console.error(...)`

Reply:
the reason for these is that I couldn't block the popup (stop the webln injection) based on the blocklist.

Reply:
For these consoles? Then it's debugging code which will be removed later? ok